frame/bags-list/src/mock.rs

```diff
@@ -148,7 +148,7 @@ impl ExtBuilder {
 	pub fn build_and_execute(self, test: impl FnOnce() -> ()) {
 		self.build().execute_with(|| {
 			test();
-			List::<Runtime>::try_state().expect("Try-state post condition failed")
+			List::<Runtime>::do_try_state().expect("Try-state post condition failed")
```

Comment:
```suggestion
			List::<Runtime>::do_try_state().expect("do_try_state post condition failed")
```

Reply:
Ideologically it is still `try-state`, `do_try_state` is just a helper. But why not